crates/torii/core/src/engine.rs

```diff
@@ -119,67 +119,66 @@ impl<'db, P: Provider + Sync> Engine<'db, P> {
         Ok(latest_block_number)
     }
 
-    pub async fn sync_range(&mut self, mut from: u64, to: u64) -> Result<()> {
+    pub async fn sync_range(&mut self, from: u64, to: u64) -> Result<()> {
```

Comment:
we may actually expose the chunk size. This will allow configuration also based on which node is used, and better tuning of the events retrieval.

Reply:
mhh i wonder if its gonna be useful as we will introduce the torii_getTransactions long poll api. so we won't always poll by events